x-pack/test/alerting_api_integration/security_and_spaces/tests/actions/builtin_action_types/email.ts

```diff
@@ -278,8 +281,9 @@ export default function emailTest({ getService }: FtrProviderContext) {
               cc: null,
               bcc: null,
               subject: 'email-subject',
-              html: '<p>email-message</p>\n',
-              text: 'email-message',
+              html: `<p>email-message</p>\n<p>--</p>\n<p>This message was sent by Kibana. <a href=\"https://localhost:5601\">Go to Kibana</a>.</p>\n`,
+              text:
+                'email-message\n\n--\n\nThis message was sent by Kibana. [Go to Kibana](https://localhost:5601).',
```

Comment:
I'm guessing we can't build a FT that provides a link into the alerting app, since we have no way of validating what the emails that get sent actually contain, since we don't have an email emulator, yet.  Good to see the link to Kibana though!

Or could/should we cheat, by passing in an explicit `kibanaFooterLink` param, in a new test?

Reply:
Yeah since there's no emulator yet I went ahead and tested the params portion instead (commit: https://github.com/elastic/kibana/pull/84371/commits/99b5cad0abad5d0a0f7fd4deca38f2787ed5136d).